framework/docker/broadcaster.go

```diff
@@ -275,7 +276,8 @@ func (b *broadcaster) BroadcastMessages(ctx context.Context, signingWallet types
 	if err != nil {
 		return sdk.TxResponse{}, err
 	}
-	b.t.Logf("broadcasted tx hash: %s", respWithTxHash.TxHash)
+	msgType := reflect.TypeOf(msgs[0]).Elem().Name()
```

Comment:
The comment was that this fn allows you to broadcast multiple msgs, but this only logs the first. Ideally we display a all of the messages being broadcast.

Reply:
no you can send multiple types of messages, eg a MsgSend, a MsgTransfer etc. There's no requirement that each implementation is the same